package.sh

```diff
@@ -10,7 +17,12 @@ TARGETDIR="rfcat_$DATESTAMP"
 TARGETPATH="../$TARGETDIR"
 echo "=== make archive ==="
 rm -rf $TARGETPATH
-hg archive -r tip $TARGETPATH
+if ( which hg )
```

Comment:
wow, you point out something even more upsetting:  this is no longer an hg repo...

Reply:
Ran into it inadvertently. Was looking at getting some things working that I haven't messed with in while since I recently reinstalled linux and didn't have some stuff installed. There are some more references to Mercurial, maybe in the wiki.